lib/membership_comparison/position_comparison.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require_relative './field_comparison'
+
+class MembershipComparison
+  class PositionComparison < FieldComparison
+    self.field = :position
+
+    def conflict
+      return 'blank superclass position' if initial_state == :partial
+      return 'superclass position matches' if initial_state == :exact
+
+      'position conflict'
+    end
+
+    private
+
+    # Unless the initial state is `ignore` we some expect this comparison to be
```

Comment:
I think there's a typo here, but I'm not sure if the "some" is meant to be a different word.